pyalgotrading/algobulls/connection.py

```diff
@@ -398,26 +443,89 @@ def get_report(self, strategy_code, trading_type, report_type, render_as_datafra
         assert isinstance(render_as_dataframe, bool), f'Argument "render_as_dataframe" should be a bool'
         assert isinstance(show_all_rows, bool), f'Argument "show_all_rows" should be a bool'
         # assert (broker is None or isinstance(broker, AlgoBullsSupportedBrokers) is True), f'Argument broker should be None or an enum of type {AlgoBullsSupportedBrokers.__name__}'
-        response = self.api.get_reports(strategy_code=strategy_code, trading_type=trading_type, report_type=report_type, country=country)
-        if response.get('data'):
+
+        if report_type is TradingReportType.PNL_TABLE:
+            response = self.api.get_reports(strategy_code=strategy_code, trading_type=trading_type, report_type=report_type, country=country, current_page=1)
+            main_data = response.get("data")
+        else:
```

Comment:
what report types are part of the else? Both stats and order history?

Reply:
`PNL_TABLE` and `ORDER_HISTORY` are the 2 report types available here